carta/session.py

```diff
@@ -358,8 +358,8 @@ def set_starting_directory(self, path):
 
     # IMAGES
 
-    @validate(String(), String(r"\d*"))
-    def open_image(self, path, hdu=""):
+    @validate(String(), String(r"\d*"), Boolean(), NoneOr(Constant(ArithmeticExpression)))
+    def open_image(self, path, hdu="", complex=False, expression=None):
```

Comment:
Instead of making `expression` default to `None`, please make it default to `ArithmeticExpression.AMPLITUDE` -- this is no longer used to turn arithmetic on and off, so if `complex` is false you just ignore it.

The description should be updated to match:
```
(...) The default is :obj:`carta.constants.ArithmeticExpression.AMPLITUDE`.
```

The validation should be changed to match (no `NoneOr`).